### PR TITLE
add otter pic and rework partner page

### PR DESCRIPTION
**Pull request by stephaniewilkinson**
Hi All!

I reworked the partner page. My goal was to make it more engaging, and with clear info. I wanted to highlight the call to action so that non-techie nonprofit people would know how to submit their project.

Feel free to wordsmith! Some of the language may be imprecise. 

My other goal was to make it thematically similar to the other pages, so I used the same headings, buttons, and columns as the sponsor page.

### CODE REVIEW

File: partner.md [hunk 1]
```diff
@@ -1,47 +1,103 @@
...
-
-The process starts with a conversation. We'll get in touch and determine how we can best help you. It may turn out that your best solution is to use something already out there and we'll help you with that.
-- About 5-6 weeks before the event we'll find someone suitable to lead a team to build your software and put them in touch with you.
-They will meet and drill down with you and what you really need. They will confirm requirements with you and then present them to the team that will be working on your software at our event.
-They will meet virtually (Skype, Google Hangouts, etc.) with you during the event to introduce the team and to clarify any questions.
-Within one month of the event's completion we will deliver the software to you.
-
-[Email us](mailto:{{ site.data.constants.contact_email }}) or submit your project [here](https://docs.google.com/forms/d/e/1FAIpQLScfGdu_Tx5UFoQ8zC2LgMCUj46RTRmx0Sfj1I1VuTYmvzo1Zw/viewform).
+<div class="col-md-4 col-sm-4 hidden-xs" markdown="1">
+  <img class="img-responsive" src="/assets/img/otter.png" alt="ruby-by-the-bay" />
```
Comment (jtu0) [inline partner.md]: The otter's background color is different from the webpage's background color - maybe use an otter with a transparent background?

File: partner.md [hunk 2]
```diff
@@ -1,47 +1,103 @@
...
+[Suggest a project we can build for your organization.](https://docs.google.com/forms/d/e/1FAIpQLScfGdu_Tx5UFoQ8zC2LgMCUj46RTRmx0Sfj1I1VuTYmvzo1Zw/viewform)
+
+We can build software to help your organization achieve its mission. We bring software engineers together to build projects for social good. We know how sometimes bad software gets in the way of letting people do their best work. We want to help make software do what it needs to do, so you can focus on your mission.
+</div>
+
+&nbsp;
+
+<div class="col-md-6" markdown="1">
+  <div class="center">
+    <h2><div class="highlight icon icon-education-science-11"></div></h2>
```
Comment (jtu0) [inline partner.md]: These little icons are _brilliant_ and they look so good!!

File: partner.md [hunk 3]
```diff
@@ -1,47 +1,103 @@
...
+  <div class="center">
+    <h2><div class="highlight icon icon-faces-users-04"></div></h2>
+    <h5 class="center">Your involvement</h5>
+  </div>
+  In order to build you the best software possible we need your participation so we can understand your needs. We require you to commit to at least two meetings with the person leading your team in the lead up to the event.
+
+  You will also need to be available at least once during the event, ideally each day. Your team will coordinate the times. This is necessary because the team building your software will inevitably run into design decisions that will require your input in order to resolve and keep the project moving forward.
+</div>
+
+<div class="col-md-4" markdown="1">
```
Comment (jtu0) [inline partner.md]: I get the flow for moving this lower, and at the same time kind of like having it up higher so people who prefer video to text can quickly see it.  Ultimately your call!

File: partner.md (hunk 1 above)
Comment (stephaniewilkinson) [reply partner.md]: perfect! i didn't have that file but I agree that would be much better

File: partner.md (hunk 3 above)
Comment (stephaniewilkinson) [reply partner.md]: either works!

**Pull request merged**